docs/coder.md

```diff
@@ -2,10 +2,6 @@
 
 coder provides a CLI for working with an existing Coder Enterprise installation
 
-### Synopsis
-
-coder provides a CLI for working with an existing Coder Enterprise installation
-
```

Comment:
Can we have a separate commit explaining why these were removed? Odd for it to be bundled into the other commit.

Reply:
Looks like it's because we updated `cobra`. CI enforces that the docs are properly generated and always up-to-date.

Reply:
Ah, didn't realize this was generated. sg